Okt/run.sh

```diff
@@ -11,12 +11,13 @@ EXTRAFLAGS="${EXTRAFLAGS:-}"
 START_FROM_SNAPSHOT="${START_FROM_SNAPSHOT:-true}"
 
 if [ "${START_FROM_SNAPSHOT}" = "true" ]; then
+    mkdir -p ${EXCHAIND_HOME}
```

Comment:
Here you have to check if the folder already exists

Reply:
Following the logic in the Dockerfile it will not exist. However, if it exists - mkdir -p will not throw an error, and it will not create the folder. Would it be more performant with a check if it exists?